urumanifest/commit.py

```diff
@@ -109,7 +127,11 @@ def on_compress(asset: manifest.ManifestEntry, future: concurrent.futures.Future
                       for i in compressed_assets)
         for server_path, staged_asset, source_asset, cached_asset in asset_iter:
             assert server_path.parent.name
-            asset_output_path = output_path.joinpath(server_path).with_suffix(f"{server_path.suffix}.gz")
+
+            if staged_asset.flags & ManifestFlags.bundle:
+                asset_output_path = output_path.joinpath(server_path).with_suffix(f"{server_path.suffix}.tgz")
+            else:
+                asset_output_path = output_path.joinpath(server_path).with_suffix(f"{server_path.suffix}.gz")
```

Comment:
```suggestion
            output_suffix = "tgz" if staged_asset.flags & ManifestFlags.bundle else "gz"
            asset_output_path = output_path.joinpath(server_path).with_suffix(f"{server_path.suffix}.{output_suffix}")
```

Flatten the logic for easier understanding.